RocketChat/Rocket.Chat
# fix: Last message appears in extended view after deletion

## Proposed changes (including videos or screenshots)

Show last visible message on the extended sidebar instead of the deleted message.




## Steps to test or reproduce

- Open a channel
- Send a message
- Hover the message and click in the three dots
- Click in “Delete”
- Check the channel/messages list

### Expected Behavior
The message should not be visible

### Actual Behavior
The message is visible in the preview

TC-711
